spec/features/admin/notes/participant_note_spec.rb

```diff
@@ -7,7 +7,7 @@
   include ParticipantDetailsSteps
   before do
     given_there_is_a_school_that_has_chosen_fip_for_2021_and_partnered
-    and_i_am_signed_in_as_an_admin
```

Comment:
I don't think this rename was the right way to go — the user action is that they sign in.